install.sh

```diff
@@ -39,7 +39,9 @@ case "${ARCH}" in
 esac
 
 # This will allow things to work without sudo
-sudo chown -R "$(id -u)":"$(id -g)" "${CREW_PREFIX}"
+crew_folders="bin cache doc docbook etc include lib lib64 libexec man sbin share tmp var"
+for folder in $crew_folders ; do sudo chown -R "$(id -u)":"$(id -g)" "${CREW_PREFIX}"/"$folder" ; done
+sudo chown "$(id -u)":"$(id -g)" "${CREW_PREFIX}"
```

Comment:
How is this different than doing a recursive chown?

Reply:
The difference is that children directly under `CREW_PREFIX` that are not in `crew_folders` will not get `chown -R`'d. In particular, if I were to have a /usr/local/mnt directory where I mount my SD card or something, I definitely would not want `install.sh` to chown -R that.

Reply:
I get that but how do we know these folders are all inclusive?  Also, `lib64` will only work on x86_64 architecture and generate an error on other architectures.  `"${CREW_PREFIX}/$folder"` should work as well.  Extra double quotes can be removed.

Reply:
Would it be better just to ignore mounted folders and include everything else?

Reply:
I suppose it would make sense to ignore symlinks as well.

Reply:
AFAIK chown ignores symlinks by default. I don't know how to get it to ignore mounted folders.

Reply:
Changed logic to avoid lib64 when not on x86_64

Reply:
I have a bad feeling this is going to end up missing things in the future.